packages/experimental/Checkbox/src/Checkbox.types.ts

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { ColorValue } from 'react-native';
+import { FontTokens, IBorderTokens, IForegroundColorTokens, IBackgroundColorTokens } from '@fluentui-react-native/tokens';
+import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
+import type { ITextProps, IViewProps } from '@fluentui-react-native/adapters';
+
+export const checkboxName = 'Checkbox';
+
+export interface CheckboxTokens extends FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens {
+  checkboxBackgroundColor?: ColorValue;
+  checkboxBorderColor?: ColorValue;
+  checkmarkColor?: ColorValue;
+  checkmarkVisibility?: number;
+  textBorderColor?: ColorValue;
+  checkboxBorderRadius?: number;
+  checkboxMarginStart?: number;
+  checkboxMarginEnd?: number;
+  checkboxVisibility?: number;
```

Comment:
What does visibility do?

Reply:
Visibility = opacity, sometimes you have a checked checkbox that's disabled and the styles for that are the same as a checked checkbox but slightly dimmed

Reply:
Could we just call it opacity then?

Reply:
Sure. It would diverge from the old Checkbox tokens though. But I would prefer it to fall more in line with the name of the style prop it's changing.